src/controllers/user/update.rs

```diff
@@ -147,12 +147,16 @@ pub async fn regenerate_token_and_send(
                 token: email.token,
             };
 
-            state.emails.send(&email.email, email1).map_err(Into::into)
-        })?;
-
-        ok_true()
+            state
+                .emails
+                .send(&email.email, email1)
+                .map_err(Into::<BoxedAppError>::into)
```

Comment:
sending emails is still a sync operation, so I think this would need to stay within a `spawn_blocking()` block for now

Reply:
ah, great catch, thanks!

Reply:
Discarded changes.